runtime/src/main/java/org/corfudb/runtime/BootstrapUtil.java

```diff
@@ -0,0 +1,64 @@
+package org.corfudb.runtime;
+
+import java.time.Duration;
+
+import javax.annotation.Nonnull;
+
+import lombok.extern.slf4j.Slf4j;
+
+import org.corfudb.runtime.clients.BaseClient;
+import org.corfudb.runtime.clients.IClientRouter;
+import org.corfudb.runtime.clients.LayoutClient;
+import org.corfudb.runtime.clients.ManagementClient;
+import org.corfudb.runtime.clients.NettyClientRouter;
+import org.corfudb.runtime.view.Layout;
+import org.corfudb.util.NodeLocator;
+import org.corfudb.util.Sleep;
+
+/**
+ * Utility to bootstrap a cluster.
+ *
+ * <p>Created by zlokhandwala on 1/19/18.
+ */
+@Slf4j
+public class BootstrapUtil {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)